shell/components/Tabbed/index.vue

```diff
@@ -230,7 +230,7 @@ export default {
           @click.prevent="select(tab.name, $event)"
         >
           <span>{{ tab.labelDisplay }}</span>
-          <i v-if="tab.displayAlertIcon" class="conditions-alert-icon icon-error icon-lg" />
+          <i v-if="tab.displayAlertIcon || (tab.error && !tab.active)" v-tooltip="t('validation.tab')" class="conditions-alert-icon icon-warning icon-lg" />
```

Comment:
Could we abstract the logic and avoid it in the markup?
E.g. 
```ts
methods: {
// ...
  hasIcon(tab: unknown): boolean {
    return tab.displayAlertIcon || (tab.error && !tab.active)
  }
// ...
}
```

Reply:
Yeah, this one's getting a little long in the tooth.